### PR TITLE
fix(coinmarket): Clean up code from Spend/DCA/P2P features

### LINKED ISSUE #13413: Clean up code from Spend/DCA/P2P features
The Spend navigation menu is hardcoded. And as we are removing this feature. We need to remove the navigation menu item (+ translation and other related code).

Will remove DCA and P2P related code also.

Cc: @marek-andrsk

### PR DESCRIPTION
**Pull request by LukasRada**
## Description

Following changes removes all code related to Spend/DCA/P2P features in coinmarket.

## Related Issue

Resolve #13413

### CODE REVIEW

File: packages/suite/src/support/messages.ts [hunk 1]
```diff
@@ -5763,10 +5424,6 @@ export default defineMessages({
         id: 'TR_NAV_P2P',
         defaultMessage: 'Buy P2P',
     },
-    TR_NAV_SAVINGS: {
-        id: 'TR_NAV_SAVINGS',
-        defaultMessage: 'Save {cryptoCurrencyName}',
-    },
```
Comment (adderpositive) [inline packages/suite/src/support/messages.ts:5769]: Consider to delete these translations: 
`TR_NAV_SPEND`, `TR_NAV_P2P` and all translations with `TR_P2P_` and `TR_SPEND_`  prefix.

File: packages/suite/src/hooks/wallet/useCoinmarketSpend.ts [hunk 2]
```diff
@@ -1,300 +0,0 @@
-import { createContext, useContext, useEffect, useState } from 'react';
-import { desktopApi } from '@trezor/suite-desktop-api';
-import { UseCoinmarketSpendProps, SpendContextValues } from 'src/types/wallet/coinmarketSpend';
```
Comment (adderpositive) [inline packages/suite/src/hooks/wallet/useCoinmarketSpend.ts:3]: Please, delete also this file `src/types/wallet/coinmarketSpend`

**Review by adderpositive — CHANGES_REQUESTED** [review]

File: packages/suite/src/support/messages.ts (hunk 1 above)
Comment (LukasRada) [reply packages/suite/src/support/messages.ts:5769]: done

File: packages/suite/src/hooks/wallet/useCoinmarketSpend.ts (hunk 2 above)
Comment (LukasRada) [reply packages/suite/src/hooks/wallet/useCoinmarketSpend.ts:3]: done

**Review by adderpositive — APPROVED** [review]

**Pull request merged**